modules/ROOT/partials/depl-examples/ubuntu-compose/shared-setup.adoc

```diff
@@ -132,11 +132,34 @@ tag::hetzner_only_1[]
 
 We recommend using key-based authentication for ssh to access the configured server instead of using user and password. This is not only beneficial for security reasons but also because you can define the public key to be installed  during the initial server configuration.
 
-Follow the https://www.ssh.com/academy/ssh/keygen[ssh-keygen guide] to generate the required keys. We recommend, if possible, using the `ed25519` algorithm. The keys to use after generation are located in `~/.ssh`.
+All OS::
+Follow the https://www.ssh.com/academy/ssh/keygen[ssh-keygen guide] to generate the required keys. We recommend, if possible, using the `ed25519` algorithm and a passphrase for enhanced security. The keys to use after generation are located in `~/.ssh`. Name the keys during the generation process accordingly to avoid overwrite existing ones.
 
-When using Putty (Windows) to access your server, you must convert the private key generated into the `ppk` format to be usable for Putty. Read the  https://www.puttygen.com[puttygen] guide to do so.
+Use the public key::
++
+--
+* During server setup described in one of the sections below, you can use the content of the public key file to setup ssh authentication when asked.
+
+* Alternativerly, after the server has been created, you can copy the _public_ key to the server by adding the _contents_ of the generated `<key>.pub` file to the `~/.ssh/authorized_keys` file manually. This step is necessary, because the server does not have password authentication which would be required to transfer the key by command.
```

Comment:
```suggestion
* Alternatively, after the server has been created, you can copy the _public_ key to the server by adding the _contents_ of the generated `<key>.pub` file to the `~/.ssh/authorized_keys` file manually. This step is necessary, because the server does not have password authentication which would be required to transfer the key by command.
```